### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by derek-cadzow**
#### What this PR does
Removes departed employee and substitutes my own ID (Derek Cadzow) until a backfill is in place. 
(Jack: not sure I should be a maintainer over the long term - can we discuss?  Is this a good idea?)

### CODE REVIEW

**Review by jdbaldry — COMMENTED** [review]
My thoughts are that it makes sense to remove Ursula because I do not believe she will be continuing to maintain these docs.

I would not add yourself as a maintainer unless you intend to be added to the @grafana/docs-metrics team and take up the responsibilities of maintainership. Primarily, that is reviewing PRs that affect the files owned by @grafana/docs-metrics

**Comment by derek-cadzow** [comment]
Yeah, not my intent to be a maintainer for that reason -- we had thought
perhaps that I should try to get a list of which issues were assigned to
Ursula and assign them to me instead so that, when the new tech writer
arrives, we'd be able to easily re-assign them to her or him.



*Derek Cadzow*

*Manager, technical writing*
*Loki, Tempo, Mimir, Cost Management, App011y, K6*
*slack: @derek.cadzow*
*email: ***@***.*** ***@***.***>*
*phone: 252-259-6967 (m)  *

File: MAINTAINERS.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Some parts of the codebase have other maintainers, the package paths also includ
 
 - `docs`:
   - Jack Baldry - [@jdbaldry](https://github.com/jdbaldry) ([Grafana Labs](https://grafana.com/))
-  - Ursula Kallio — [@osg-grafana](https://github.com/osg-grafana) ([Grafana Labs](https://grafana.com/))
+  - Derek Cadzow — [@derek-cadzow](https://github.com/derek-cadzow) ([Grafana Labs](https://grafana.com/))
```
Comment (jdbaldry) [inline MAINTAINERS.md:35]:
```suggestion
```

**Review by jdbaldry — COMMENTED** [review]
In that case @derek-cadzow I think my suggestion does what you want and if you accept it, we can close and merge this PR.

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**